libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -2090,7 +2090,9 @@ bool ExpressionCompiler::cleanupNeededForOp(Type::Category _type, Token::Value _
 {
 	if (Token::isCompareOp(_op) || Token::isShiftOp(_op))
 		return true;
-	else if (_type == Type::Category::Integer && (_op == Token::Div || _op == Token::Mod))
+	else if (_op == Token::Div || _op == Token::Mod)
```

Comment:
Should this keep `_type == Type::Category::Integer`?

Reply:
@chriseth (I didn't write this code)
I guess I accidentally reset author (there were some issues with timestamp and github log)